src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -156,10 +162,6 @@ export default {
 	-moz-user-select: none;
 	-ms-user-select: none;
 	user-select: none;
-	display: flex;
-	flex-direction: column;
-	flex-grow: 0;
-	flex-shrink: 0;
```

Comment:
This needs reverting it heavily breaks the talk left sidebar:
![grafik](https://user-images.githubusercontent.com/213943/236216187-e1f5e804-4d7b-4430-9ef1-296af70b88ff.png)

Reply:
@JuliaKirschenheuter :warning:

Reply:
Yes, but not all of them.

These styles are flex container styles. They are moved.

```css
display: flex;
flex-direction: column;
```

while these are flex elements styles. They should be only here.

```css
flex-grow: 0;
flex-shrink: 0;
```

Reply:
> this seems absolutely not related to this PR

50% of these styles are flex container styles. In this PR there was created a new flex container for all content, and all flex container styles were moved to the new flex container. So, they are related.

Although, another 50% were flex element styles and really not related.

Reply:
Ok, this what i feared: it is hard to test every app. Thank you @ShGKme for explanation and fixing!